life_expectancy/tests/test_main.py

```diff
@@ -0,0 +1,30 @@
+from life_expectancy.main import main
+import pandas as pd
+from pytest import MonkeyPatch
+
+
+
+def test_main(
+        monkeypatch: MonkeyPatch,
+        eu_life_expectancy_raw: pd.DataFrame,
+        pt_life_expectancy_expected: pd.DataFrame,
+) -> None:
+    """test the main function result against expected"""
+
+    # Mock the inputs
+    def _mock_load_data(*arg, **kwargs) -> pd.DataFrame:
+        return eu_life_expectancy_raw
+    
+    monkeypatch.setattr("life_expectancy.loaders.pd.read_csv", _mock_load_data)
+
+    def _mock_save_data(*arg, **kwargs) -> None:
+        print("Data was saved")
+    
+    monkeypatch.setattr("life_expectancy.loaders.pd.DataFrame.to_csv", _mock_save_data)
```

Comment:
Suggestion: you can also test if the save_data was called with something like `assert out == f"Data was saved\n"`